server/src/test/java/org/opensearch/action/support/master/TransportMasterNodeActionTests.java

```diff
@@ -452,15 +454,19 @@ public void testDelegateToFailingMaster() throws ExecutionException, Interrupted
                 // reset the same state to increment a version simulating a join of an existing node
                 // simulating use being disconnected
                 final DiscoveryNodes.Builder nodesBuilder = DiscoveryNodes.builder(clusterService.state().nodes());
-                nodesBuilder.masterNodeId(masterNode.getId());
+                nodesBuilder.masterNodeId(clusterManagerNode.getId());
                 setState(clusterService, ClusterState.builder(clusterService.state()).nodes(nodesBuilder));
             } else {
                 // simulate master restart followed by a state recovery - this will reset the cluster state version
                 final DiscoveryNodes.Builder nodesBuilder = DiscoveryNodes.builder(clusterService.state().nodes());
-                nodesBuilder.remove(masterNode);
-                masterNode = new DiscoveryNode(masterNode.getId(), masterNode.getAddress(), masterNode.getVersion());
-                nodesBuilder.add(masterNode);
-                nodesBuilder.masterNodeId(masterNode.getId());
+                nodesBuilder.remove(clusterManagerNode);
+                clusterManagerNode = new DiscoveryNode(
+                    clusterManagerNode.getId(),
+                    clusterManagerNode.getAddress(),
+                    clusterManagerNode.getVersion()
+                );
+                nodesBuilder.add(clusterManagerNode);
+                nodesBuilder.masterNodeId(clusterManagerNode.getId());
```

Comment:
masterNodeId -> clusterManagerNodeId?

Reply:
I will rename masterNodeId() in the next phase, which is mentioned in issue https://github.com/opensearch-project/OpenSearch/issues/1684.